src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/RSA.cs

```diff
@@ -59,6 +59,30 @@ public static RSA Create(RSAParameters parameters)
             }
         }
 
+        /// <summary>
+        ///   The maximum number of bytes an RSA operation can produce.
+        /// </summary>
+        /// <remarks>
+        ///   The maximum output size is defined by the RSA modulus, or key size. The key size, in bytes, is the maximum
+        ///   output size. If the key size is not an even number of bytes, then it is rounded up to the nearest number of
+        ///   whole bytes for purposes of determining the maximum output size.
+        /// </remarks>
+        /// <exception cref="CryptographicException">
+        ///   <see cref="AsymmetricAlgorithm.KeySize" /> returned a value that is not a possible RSA key size.
+        /// </exception>
+        public int GetMaxOutputSize()
+        {
+            if (KeySize <= 0)
+            {
+                throw new CryptographicException(SR.Cryptography_InvalidKeySize);
+            }
+
+            // KeySize is in bits. Add 7 to round up to nearest byte, then divide by 8.
```

Comment:
Adding 7 doesn't really round up to the nearest byte :)

I'd say something more like  "KeySize is in bits, but since one extra bit would require a whole byte to write to, it's ceil(KeySize / 8).  Rather than using floating point, we can get the same output with `(KeySize + 7) / 8` under integer math."

But with a few more editorial passes to get it smaller.

So maybe just

```suggestion
            // KeySize is in bits. Add 7 before dividing by 8 to get ceil() instead of floor().
```